### PR TITLE
v1: add gunk definitions for counterparty metadata

### PR DESCRIPTION
**Pull request by jwangsadinata**
This commit adds the gunk definitions for the counterparty metadata.

### CODE REVIEW

File: v1/counterpartymetadata/counterpartymetadata.gunk [hunk 1]
```diff
@@ -0,0 +1,2186 @@
...
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type DeletePrivateAliasRequest struct {
+	// ID is the private_alias id that will be deleted.
+	ID string `pb:"1" json:"alias"`
```
Comment (nkristianto) [inline v1/counterpartymetadata/counterpartymetadata.gunk]: ups, wrong json tag

File: v1/counterpartymetadata/counterpartymetadata.gunk [hunk 2]
```diff
@@ -0,0 +1,2186 @@
...
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type GetPrivateAliasRequest struct {
+	// ID is the private_alias unique identifier.
+	ID string `pb:"1" json:"alias"`
```
Comment (nkristianto) [inline v1/counterpartymetadata/counterpartymetadata.gunk]: this one also

**Review by nkristianto — APPROVED** [review]
nit, offer all LGTM

File: v1/counterpartymetadata/counterpartymetadata.gunk (hunk 1 above)
Comment (jwangsadinata) [reply v1/counterpartymetadata/counterpartymetadata.gunk]: thank you thank you, that one does not need to be there actually.

File: v1/counterpartymetadata/counterpartymetadata.gunk (hunk 2 above)
Comment (jwangsadinata) [reply v1/counterpartymetadata/counterpartymetadata.gunk]: will fix

Comment (jwangsadinata) [reply v1/counterpartymetadata/counterpartymetadata.gunk]: updated accordingly.

File: v1/counterpartymetadata/counterpartymetadata.gunk [hunk 3]
```diff
@@ -0,0 +1,2138 @@
...
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type UpdatePublicAliasRequest struct {
+	// PublicAlias is the public_alias metadata made on a other_account.
+	PublicAlias PublicAlias `pb:"1" json:"alias"`
```
Comment (ghazninattarshah) [inline v1/counterpartymetadata/counterpartymetadata.gunk:561]: json tag shall be `public_alias`

Comment (ghazninattarshah) [reply v1/counterpartymetadata/counterpartymetadata.gunk:561]: I am not sure if you want to differentiate `public` or `private` alias on json tags.

File: v1/counterpartymetadata/counterpartymetadata.gunk [hunk 4]
```diff
@@ -0,0 +1,2138 @@
...
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetOtherAccountMetadataResponse struct {
+	PublicAlias       string            `pb:"1" json:"public_alias"`
+	PrivateAlias      string            `pb:"2" json:"private_alias"`
```
Comment (ghazninattarshah) [inline v1/counterpartymetadata/counterpartymetadata.gunk:708]: May be we should match `public_alias` and `private_alias` json tag all over the structs, few places it was mentioned just as `alias`

**Review by ghazninattarshah — APPROVED** [review]
LGTM

**Pull request merged**